packages/ag-charts-enterprise/project.json

```diff
@@ -126,6 +126,14 @@
       "options": {
         "commands": ["node tools/package/sanity-check-package.js dist/packages/contents/ag-charts-enterprise/package"]
       }
+    },
+    "validate-size": {
```

Comment:
Nit: Could we have this as `test:validate-size` and link it from the `test` target perhaps?

Should we also have this in the `ag-charts-community` package too?